build.cake

```diff
@@ -0,0 +1,32 @@
+var target = Argument("target", "Build");
+
+Task("Clean")
+	.Does(() => 
+{
+	var settings = new DotNetCoreCleanSettings
+     {
+         Framework = "netstandard2.0",
+         Configuration = "Debug",
+         OutputDirectory = "./artifacts/"
+     };
+
+	 DotNetCoreClean("./src/CodingMilitia.PlayBall.Shared.StartupTasks", settings);
+	
+});
+
+Task("Build")
+	.IsDependentOn("Clean")
+	.Does(() => 
+{
+	var settings = new DotNetCoreBuildSettings
+     {
+         Framework = "netstandard2.0",
+         Configuration = "Debug",
```

Comment:
Why debug?

Reply:
`Debug` should be the configuration and passed as an argument. for now... YAGNI

Reply:
Ok, for now there's no problem. But we must remember to change it when we start packing NuGets.